### PR TITLE
Northern Britain and Picts Overhaul

### PR DESCRIPTION
**Pull request by MaelgwnGwynedd**

### CODE REVIEW

File: WTWSMS/common/religions/00_religions.txt [hunk 1]
```diff
@@ -367,9 +367,11 @@ christian = {
 		can_grant_claim = yes
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
```
Comment (TurtleShroom) [inline WTWSMS/common/religions/00_religions.txt]: Why did you change the wives here? Why don't Insular Christians have three wives?

File: WTWSMS/common/religions/00_religions.txt [hunk 2]
```diff
@@ -1187,6 +1189,46 @@ christian = {
...
+		priests_can_inherit = no
+		max_wives = 1
+		
+		can_retire_to_monastery = yes
+		parent = insular
+		bs_marriage = yes
+		pc_marriage = yes
+		religious_clothing_head = 1
+		religious_clothing_priest = 1
+
```
Comment (TurtleShroom) [inline WTWSMS/common/religions/00_religions.txt]: Why do Belanists commit incest and dress like Insular Christians?

File: WTWSMS/common/religions/00_religions.txt [hunk 3]
```diff
@@ -2516,12 +2558,13 @@ pagan_group = {
...
 		feminist = yes
 		defensive_attrition = yes
-		max_consorts = 3
 		peace_prestige_loss = yes # Lose Prestige while at peace
 		raised_vassal_opinion_loss = no
 		allow_looting = yes
 		reformed = celtic_pagan_reformed
 		aggression = 1.5 # AI aggression factor
+		bs_marriage = yes
+		pc_marriage = yes
```
Comment (TurtleShroom) [inline WTWSMS/common/religions/00_religions.txt]: Why are unreformed Celtic Pagans incestous? -and if they are incestors, why aren't the Reformed Celtic Pagans also incestous?

**Comment by MaelgwnGwynedd** [comment]
-They didn't practice polygamy. And it's one wife, not two wives.
-Because they're still Christians. Incest was relatively taboo amongst the insulars but not to people of looser faiths like paganism or heresies (De Excidio by Gildas)
-See above. Reformed Celtic Pagans never existed, so I can't answer that.

**Comment by MaelgwnGwynedd** [comment]
Those aren't even on this pull request, though.

File: WTWSMS/common/religions/00_religions.txt (hunk 1 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: They didn't practice polygamy. And it's one wife, not two wives.

File: WTWSMS/common/religions/00_religions.txt (hunk 2 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: Because they're still Christians. Incest was relatively taboo amongst the insulars but not to people of looser faiths like paganism or heresies (De Excidio by Gildas)

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: See above. Reformed Celtic Pagans never existed, so I can't answer that.

**Comment by TurtleShroom** [comment]
"Max wives" covers ADDITIONAL wives. If you want to keep them from polygamy, you have to remove that line entirely. Right now, the code creates two wives.
 
Why did Insular Christians have polygamy in this Mod? What about concubinage?

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (TurtleShroom) [inline WTWSMS/common/religions/00_religions.txt]: Shouldn't Reformed Celtics have the sake  incest ability as normal Celtics?

**Comment by MaelgwnGwynedd** [comment]
I don't anticipate EVERY change. If you want them to have incest, add it.

I don't know why Insular Christians have polygamy in the Mod, all I know is it's wrong. Why do African Warlords have Denmark or why is Fib called "Taezuli"?

**Comment by MaelgwnGwynedd** [comment]
Also, they aren't strange additions.

File: WTWSMS/common/religions/00_religions.txt (hunk 1 above)
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: According to input we received they did practice it.

File: WTWSMS/common/religions/00_religions.txt [hunk 4]
```diff
@@ -367,9 +367,11 @@ christian = {
 		can_grant_claim = yes
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
 		
 		can_retire_to_monastery = yes
+		bs_marriage = yes
+		pc_marriage = yes
```
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: Did they practice incest?

File: WTWSMS/common/religions/00_religions.txt (hunk 2 above)
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: That makes sense, but there is a difference from it being possible and actively encouraged.

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: See above on incest and concubines.

**Comment by MaelgwnGwynedd** [comment]
What input? It's wrong.

File: WTWSMS/common/religions/00_religions.txt (hunk 2 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: True, but it seemed backwards not to allow it.

File: WTWSMS/common/religions/00_religions.txt (hunk 1 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: That's wrong, what input?

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: They did practice incest.

Maelgwn Gwynedd married his cousin Gwallwyn (Gildas' De Excidio)
Geraint ap Erbin married his unnamed niece (Harleian Genealogies)
Gworthigern (Vortigern) married his daughter
Vortipor married his sister and had incestuous relations with his daughter (Gildas' De Excidio)

I don't think it's not too much of a stretch to allow pc and bs incest for everyone.

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: Oh, Maelgwn also then went on to marry his nephew's wife. While not blood incest, this was *probably* (and this IS conjectural) political incest, given the Celts' views on the family as a single unit, his niece had become "part of the family" despite not being part of it by blood.
If you see what I mean?

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: This whole attitude may have been the faults of the Romans, who had very relaxed views on incest (Caligula married his sister and declared her his Goddess sister-wife). It was not actually a crime, but was often used in a slanderous manner.

I can confirm neither of the four men above were slandered for their choice of partner (quasi-nuclear relatives) but only by their other actions. Gildas casually mentions Vortipor's incest then says he's evil since he usurped the throne.

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: If you believe this is still insufficient grounds to add it, though, could we maybe compromise in some way? Since there is clearly historical evidence of nuclear incest which was quasi-accepted.

File: WTWSMS/common/religions/00_religions.txt (hunk 2 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: See below.

File: WTWSMS/common/religions/00_religions.txt (hunk 4 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: See my other comments.

Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: What other comment? I posted below on incest.

**Comment by MaelgwnGwynedd** [comment]
What? Just read what I posted.

File: WTWSMS/common/religions/00_religions.txt (hunk 1 above)
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: I don't remember Enlil's answer. Will see if I can find it.

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: It being allowed makes sense, although I'm worried that the AI would start prioritising it.

**Comment by loup99** [comment]
Never mind.

File: WTWSMS/common/religions/00_religions.txt (hunk 1 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: Thanks. I think they may have based it on David Hughes' fictional "Chronicles of Britain".

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: Somewhat how like Xwedodah-style faiths prioritise sister-wives for rulers? That could be a problem.

**Comment by MaelgwnGwynedd** [comment]
Right

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: Yes. But I'm not sure if it is another weight that makes that important or if it tied to enabling it.

File: WTWSMS/common/religions/00_religions.txt (hunk 1 above)
Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: This? https://www.amazon.fr/British-Chronicles-David-Hughes/dp/0788433016

Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: I found the comment:

> In early Ireland polygamy was practiced using the justification that it was in the old testament.

**Comment by MaelgwnGwynedd** [comment]
Adding some new commits.
-Danish and Dano-Jutish legacies
-Corrected Saxony, Jutland and Angeln
-Renamed them accordingly
-Added the Danish kings who took over Angeln and Jutland
-Added the Anglian (Slevsig and Fyn) and Jutish (Jylland) Kings

File: WTWSMS/common/religions/00_religions.txt (hunk 1 above)
Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: There's nothing to substantiate that. Hell, the earliest Irish laws stress that a man has only one wife and that any lovers/mistresses he may have don't enjoy the same rights (such as divorce if he should beat her, be accused of adultery or a homosexual).

Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: So you would say neither polygamy nor concubinage?

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: Also, I find it odd they'd ever use a bible as justification when for several centuries the Irish took pride in being insulars. The "Romanii", a faction of Irish monks in the 800's who advocated following the Roman Catholics a lot more, wanted to put an end to incestuous marriage between cousins, and the Irish told them to, well, fuck off.

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: There's SOME evidence of the Ancient Gaels (Not at ALL the Ancient Britons) had some sort of Polygamy. But we're talking about the 1000's BC here, when they were still glorified hunter gatherers, and the Irish chieftain may have had a harem. That was long dead when the Romans arrived.

Though it's true the Irish practiced what you may call "pseudo-polygamy". They very often divorced (or killed) their wives for dumb reasons just to get a new one, which you can easily see when some Irish kings had six or seven wives in succession. Think of situations like the later Henry VIII. But even _that_ pseudo-polygamy is later period than the mod.

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: "So you would say neither polygamy nor concubinage?"

Definitely. The Christian "lovers" system is absolutely perfect.

Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: And concubines?

Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: OK.

File: WTWSMS/common/religions/00_religions.txt (hunk 3 above)
Comment (TurtleShroom) [inline WTWSMS/common/religions/00_religions.txt]: IIRC, they only priorotize it if it is a Xaowada union or it is the most prestigious choice.

Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: If it does that it is inappropriate. From what I understand these marriages happened but were not a priority or something which was desired.

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: Yes. Can we override the priority?

Comment (MaelgwnGwynedd) [inline WTWSMS/common/religions/00_religions.txt]: I thought only the divine_bloodline = yes functioned like that.

Comment (loup99) [inline WTWSMS/common/religions/00_religions.txt]: The wiki seems to agree with you: http://www.ckiiwiki.com/Religion_modding

**Comment by MaelgwnGwynedd** [comment]
How long until you're done reviewing?

**Comment by loup99** [comment]
Depends. The map overhaul and melting-pots are higher priorities for me.

Do you know if any DLC was used for the new portraits?

**Comment by MaelgwnGwynedd** [comment]
Celtic DLC, of course.

**Comment by MaelgwnGwynedd** [comment]
What do you mean the melting-pots?

**Comment by loup99** [comment]
OK, then that means I will have to do some modifications to what you currently have. As the files are now, we can't distribute them. 

The melting-pots as in this: https://en.wikipedia.org/wiki/Melting_pot

**Comment by MaelgwnGwynedd** [comment]
Why? It can only be accessed by people who have the DLC.

**Comment by loup99** [comment]
See the sub-mods

**Comment by MaelgwnGwynedd** [comment]
Ok, is the rest of the branch fine?

**Comment by loup99** [comment]
That is what I'm going to look at before merging.

**Comment by MaelgwnGwynedd** [comment]
But if I keep committing more and more, this will never end.

**Comment by loup99** [comment]
Well, the map overhaul isn't getting done for a long while. But the British Isles will be our step once we are done with Egypt and Hispania. So we will determine exactly how we integrate it depending on the scale of the changes. But don't worry, nothing will get lost, and everything which isn't integrated will be discussed thoroughly beforehand.

**Comment by MaelgwnGwynedd** [comment]
Good.

**Comment by MaelgwnGwynedd** [comment]
New additions.

**Comment by loup99** [comment]
The warrior government has already been discussed, so I refer to that for my thoughts.

**Comment by MaelgwnGwynedd** [comment]
Ok

**Comment by MaelgwnGwynedd** [comment]
Don't push this. You're free to overhaul it yourself if you're not going to understand this push is exactly that.

**Comment by loup99** [comment]
If you prefer this to not be included into the main mod at all rather than after the map overhaul, I respect that.

**Comment by loup99** [comment]
Also, it is "yourselves", as in all developers and all of those submitting feedback. Thought it was important to mention as it is an interactive project.

**Comment by MaelgwnGwynedd** [comment]
No, no it isn't interactive.

**Comment by MaelgwnGwynedd** [comment]
There's no need for a British map overhaul, the borders are *not wrong*.

**Comment by loup99** [comment]
The borders in the release version are wrong. On this former pull-request, you worked on improving that setup, but you didn't add any province. Our aim is to add provinces and look over the history. Once that is done I had planned to merge the two overhauls.

**Comment by MaelgwnGwynedd** [comment]
They are NOT wrong in my overhaul.

**Comment by MaelgwnGwynedd** [comment]
Use this if you want, but I don't want to be a part of this. Sorry.

**Comment by MaelgwnGwynedd** [comment]
Bring overruled, saying my work is basically worthless, refusing to
implement it now or then and not allowing me to communicate with anyone or
the mod, is not on.

**Comment by loup99** [comment]
Maelgwyn, unfortunately I can't be everywhere all the time. Modding is only a hobby, and if no one can review the content we can't integrate it. I don't know where you read that your work was worthless or that I refused anything, but that was not my intent in case it was how you felt.

**Comment by MaelgwnGwynedd** [comment]
"We won't use it because of the Operation Ptolemy changes we will do, not
your correct and already done work."
"This is a team effort." and then "This is not up for discussion"

You've also refused to discuss important things.

**Comment by loup99** [comment]
Also, who has limited you from communicating? That is not something I'm aware of.

**Comment by MaelgwnGwynedd** [comment]
You had me banned from the forum where the mod and Operation Ptolemy are
being worked on. I didn't even know about any map overhaul until now, seven
months in.

**Comment by loup99** [comment]
> "We won't use it because of the Operation Ptolemy changes we will do, not
> your correct and already done work."

Operation Ptolemy is, if you missed it, an operation about overhauling the entire province map. Some areas we are less happy with, others more. The British Isles are highly requested, so it is an area we will suggest changes to. I don't see how this conflicts with your work, other than that both will have to be merged.  Nothing is getting thrown into a rubbish bin or just discarded outright, if there is anything we don't keep it will be discussed, and everything we touch in any way will be on commits which can be commented upon. 

What is not up for discussion is to not do Operation Ptolemy, as it has already been started and there is a consensus around it. 

I also have a tendency to avoid repeating things, it becomes tedious for me and I don't have enough time to constantly make sure every single line of code is reviewed. This means that there will be delays.

**Comment by loup99** [comment]
> You had me banned from the forum where the mod and Operation Ptolemy are being worked on.

This has already been addressed and closed.

> I didn't even know about any map overhaul until now, seven months in.

You were initially involved in looking at provinces for the British Isles, before you decided to leave the forum.

**Comment by TurtleShroom** [comment]
DUDE! He is NOT forsaking your work! HE IS GOING TO MERGE YOUR CHANGES WITH THE OVERHAUL! Stop acting like he is shutting you down.

**Comment by TurtleShroom** [comment]
He WILL include your work AFTER the reconfiguration. He is NOT saying your work is useless. It WILL be in the Mod.

**Comment by MaelgwnGwynedd** [comment]
MY MAP IS FUCKING CORRECT.

**Comment by MaelgwnGwynedd** [comment]
There's no need, since the provinces are correct in my changes. There's no
need to overhaul Britain.

**Comment by loup99** [comment]
The discussion ends here. MaelgwnGwynedd prefers to not be a part of this. This isn't a place for heated debate.
